packages/vm/core/testcore/accounts_test.go

```diff
@@ -481,9 +481,9 @@ func initDepositTest(t *testing.T, initCommonAccountBaseTokens ...coin.Value) *a
 	ret := &accountsDepositTest{}
 	ret.env = solo.New(t, &solo.InitOptions{Debug: true, PrintStackTrace: true})
 
-	ret.chainOwner, ret.chainOwnerAddr = ret.env.NewKeyPairWithFunds(ret.env.NewSeedFromIndex(10))
+	ret.chainOwner, ret.chainOwnerAddr = ret.env.NewKeyPairWithFunds(ret.env.NewSeedFromTestName(t.Name()))
 	ret.chainOwnerAgentID = isc.NewAddressAgentID(ret.chainOwnerAddr)
-	ret.user, ret.userAddr = ret.env.NewKeyPairWithFunds(ret.env.NewSeedFromIndex(11))
+	ret.user, ret.userAddr = ret.env.NewKeyPairWithFunds(ret.env.NewSeedFromTestName(t.Name()))
```

Comment:
I guess here `ret.chainOwner` and ret.user` would be the same? They are hash value from the same string

Reply:
No, it hashes the function name and a timestamp internally. It's quite opaque. I'll rename the function.